src/NosCore.GameObject/Character.cs

```diff
@@ -87,6 +114,36 @@ const int
 
         public byte Equipment { get; set; }
 
+        public void LoadRelations()
+        {
```

Comment:
this doesnt require a method on character just put it on the loading for first connection on character